tests/unit/test_versioning.py

```diff
@@ -0,0 +1,69 @@
+import pytest
+from flask import Flask, Response
+
+from app.versioning import versioned, LATEST_API_VERSION, InvalidApiVersion
+
+
+@pytest.mark.parametrize('versions', ['1.4', '1.4.1-snapshot'])
```

Comment:
This is interesting. We changed `VALID_API_VERSIONS` to a tuple, but it still works when you pass in a list. I think the API for lists is similar enough to the one for tuples that it doesn't really matter which you pass in. Do you think this is fine or should we change it to a tuple just to provide a code example that shows what you'll actually want to pass in if/when you implement this in the main app logic?

Reply:
If/when I fix my other nitpick above, I'm going to change this too